File: Leetcode/Day-1/Python/2427. Number of Common Factors.py
class Solution:
    def commonFactors(self, a: int, b: int) -> int:
        if a <= b:
            last = a
        else:
            last = b

        count = 0
        for i in range(1, last+1):
            if a % i == 0 and b % i == 0:
                count += 1
        
        return count

File: Leetcode/Day-1/Java/2427. Number of Common Factors.java
class Solution {
    public int commonFactors(int a, int b) {
        int count = 0;
        int last;
        if (a <= b) last = a;

        else last = b;
        
        for (int i = 1; i <= last; i++){
            if (a % i == 0 && b % i == 0){
                count += 1;
            }
        }
        return count;
    }
}
